packages/daemon/src/runner/job-runner.ts

```diff
@@ -57,8 +57,11 @@ export interface JobResult {
  */
 export class JobRunner {
   opts: RunnerOptions;
+  tmpdataDir: string;
+
   constructor(opts: RunnerOptions) {
     this.opts = opts;
+    this.tmpdataDir = path.join(this.opts.runnable.workingDir, 'tmpdata');
```

Comment:
The `JobRunner` should not operate on the file system, may I ask you to move this to costa's Runnable?

Reply:
done